metricbeat/module/kubernetes/state_persistentvolumeclaim/_meta/test/ksm.v2.10.0.plain.expected

```diff
@@ -6,6 +6,9 @@
 		},
 		"MetricSetFields": {
 			"access_mode": "ReadWriteOnce",
+            "created": {
```

Comment:
Isn't this file created automatically ? I wonder why indentation is wrong.

Reply:
> Isn't this file created automatically ? I wonder why indentation is wrong.

Sorry, It is very likely that I did something wrong during my tests, I'm still trying to familiarize myself with the unit tests in this package. Could you explain me what command is supposed to generate this file? 🙇